.github/workflows/tests.yml

```diff
@@ -1,57 +1,59 @@
 # https://help.github.com/en/categories/automating-your-workflow-with-github-actions
 
 on:
-    - pull_request
-    - push
+  push:
+    branches:
+      - master
+  pull_request:
```

Comment:
I decided to keep it on the safe side so this means it runs:
- on pull requests
- on pushes but only to the `master` branch